src/index.html

```diff
@@ -65,5 +70,214 @@ <h1 class="headline__title">
+        <img
+          class="project-card__image"
+          src="./assets/images/snapshoot-3.jpg"
+          alt="Snapshoot 1"
+        />
+        <div class="project-card__body">
+          <div class="project-card__header">
+            <h2 class="project-card__header-title">Tonic</h2>
+            <div class="project-card__header-subtitle">
+              <span>CANOPY</span>
+              <span>Back End Dev</span>
+              <span>2015</span>
+            </div>
+          </div>
+          <p class="project-card__description">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="project-card__tags-list">
+            <li class="project-card__tags-list-item">html</li>
+            <li class="project-card__tags-list-item">css</li>
+            <li class="project-card__tags-list-item">javascript</li>
+          </ul>
+          <div class="project-card__actions">
+            <button class="button" type="button">See Project</button>
+          </div>
+        </div>
+      </div>
+      <div class="project-card">
+        <img
+          class="project-card__image"
+          src="./assets/images/snapshoot-4.jpg"
+          alt="Snapshoot 1"
+        />
+        <div class="project-card__body">
+          <div class="project-card__header">
+            <h2 class="project-card__header-title">Multi-Post Stories</h2>
+            <div class="project-card__header-subtitle">
+              <span>CANOPY</span>
+              <span>Back End Dev</span>
+              <span>2015</span>
+            </div>
+          </div>
+          <p class="project-card__description">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="project-card__tags-list">
+            <li class="project-card__tags-list-item">html</li>
+            <li class="project-card__tags-list-item">css</li>
+            <li class="project-card__tags-list-item">javascript</li>
+          </ul>
+          <div class="project-card__actions">
+            <button class="button" type="button">See Project</button>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="about">
+      <div class="about-me">
+        <div class="about-me__body">
+          <div class="about-me__title">
+            <h2>
+              About <br />
+              Myself
+            </h2>
+          </div>
+          <div class="about-me__text">
+            <p>
+              Hello I’m a software developer! I can help you build a product ,
+              feature or website Look through some of my work and experience! If
+              you like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </p>
+          </div>
+          <div class="about-me__actions">
+            <a class="link" href="#">LET’S CONNECT</a>
```

Comment:
- [ ] I think we should make this `<a>` tag a `<h3>` or `<h4>` tag. It's not actually a link so it's kind of misleading and will cause problems for screen readers.

Reply:
Oh! I thought that it's a link that takes to the contact form.

Reply:
### Thank you for your comment.
issue fixed here [Fix headline and about subtitles](https://github.com/teckim/portfolio/pull/2/commits/7132c51f661d68ba5ba98574bc9b248b8feedd5e)